ui/components/ui/unit-input/unit-input.component.js

```diff
@@ -84,7 +88,12 @@ export default class UnitInput extends PureComponent {
     let value = userInput;
 
     if (userInput.length && userInput.length > 1) {
-      value = removeLeadingZeroes(userInput);
+      value = removeLeadingZeroes(userInput).replace(/,/u, '.');
```

Comment:
what if the user enters "2,500" and wants to send 2500?

Reply:
Only possible for pasting which should be pretty rare for this type of input. Also, in my opinion, I think it's better to low ball than high ball (i.e., remove the comma all together)